tools/create_lmdb_dataset.py

```diff
@@ -5,6 +5,17 @@
 import numpy as np
 import io
 from PIL import Image
+import sys
```

Comment:
All datasets are in LMDB format and can be created without modifying create_lmdb_dataset.

Reply:
I used this for trying out some stuff locally, so I forgot to remove it. Thanks.